src/components/Message/Message.js

```diff
@@ -5,17 +5,37 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Typography } from '@material-ui/core';
+import classNames from 'classnames';
+import { Typography, withStyles } from '@material-ui/core';
 
-const Message = ({ children }) =>
-  children ? <Typography variant="subheading">{children}</Typography> : null;
+import MessageStyles from './Message.style';
+
+const Message = ({ children, classes, type }) =>
```

Comment:
I adjusted the `Message` component so that it can return an error or info messages. More message types are likely to be needed in the future, so I prepared for that :)